zebrad/tests/acceptance.rs

```diff
@@ -522,7 +522,7 @@ fn version_no_args() -> Result<()> {
 
     let testdir = testdir()?.with_config(&mut default_test_config()?)?;
 
-    let child = testdir.spawn_child(args!["version"])?;
+    let child = testdir.spawn_child(args!["--version"])?;
```

Comment:
Removing the version command is a breaking change so there should be a changelog note for it.

As an alternative, add a version command that calls --version.

Reply:
> Removing the version command is a breaking change so there should be a changelog note for it.

I'll add something to the changelog for now.

> As an alternative, add a version command that calls --version.

A suggestion PR would be very welcome.

Reply:
Fixing a breaking change that most users won't notice isn't a priority for me right now, I still have some stable release blockers to implement.

Feel free to open a ticket, but if it doesn't get done by the first stable release we should just close that ticket.

Reply:
Opened issue https://github.com/ZcashFoundation/zebra/issues/6823 to optionally follow up on this.